src/App.js

```diff
@@ -1,24 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Based on the requirements of this project, the App component should be class-based.

Reply:
Yeah thank you I forgot about that :+1: